rebasehelper/options.py

```diff
@@ -146,6 +146,15 @@
         "dest": "non_interactive",
         "help": "do not interact with user",
     },
+    {
+        "name": ["--strategy-option"],
+        "choices": ["ours", "theirs"],
```

Comment:
I don't like this too much. Although this is close to actual git arguments, I think it would be more user friendly to call the option something like `--favor-on-conflict` with choices `upstream` and `downstream`.